docs/UserGuide.md

```diff
@@ -1,134 +1,340 @@
+
+Listing tasks with a certain importance. <br>
+Format: `list i/[IMPORTANT]`
+
+By keying in the following, it will display a list of the tasks associated with the input importance.
+
+> list i/[IMPORTANCE]
+
+> acceptable formats of IMPORTANT input: 
+>     1,2,3 -- where 1 means the most important tasks
+>     red, yellow, green
+
+<br><br>
+
+#### <a id="find"></a>Finding all tasks containing any keyword in their name: `find`
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+The Find command will search all tasks which contain any of the given keywords.
+
+> find KEYWORD [MORE_KEYWORDS]
+
+> * The search is case sensitive. e.g `homework` will not match `Homework`
+> * The order of the keywords does not matter. e.g. `Movie Night` will match `Night Movie`
 > * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * Only full words will be matched e.g. `visit` will not match `visiting`
+> * Tasks matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `Concert` will match `Michael Jackson Concert`
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find meeting`<br>
+  Returns `Lunch meeting` but not `Lunch Meeting`
+* `find Fish Friday`<br>
+  Returns Any task having containing `Fish`, or `Friday` (eg. ‘Visit Fish Market’ and ‘Listen to Rebecca Black’s Friday’)
+
+<br><br>
+
+#### <a id="edit"></a>Editing a task: `edit`
+Format: `edit INDEX [n/NAME], [s/START_DATE], [st/START_TIME], [e/END_DATE], [st/END_TIME], [i/IMPORTANCE]`
+
+Made a spelling mistake or your event was postponed? You can use the Edit Command to swiftly rectify any tasks on the task list.
+
+This can be done by typing the following:
+
+> edit INDEX [n/NAME], [s/START_DATE], [st/START_TIME], [e/END_DATE], [et/END_TIME], [i/IMPORTANCE]
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Parameters: <br>
+INDEX: the number that was linked to the task you wish to edit <br>
+[n/NAME]: the name you wish to change to <br>
+[s/START_DATE]: the start date you wish to change to <br>
+[st/START_TIME]: the start time you wish to change to <br>
+[e/END_DATE]: the end date you wish to change to <br>
+[et/END_TIME]: the end time you wish to change to <br>
+[i/IMPORTANCE]: the level of priority you wish to change to
 
-> Deletes the person at the specified `INDEX`. 
+Other than the INDEX which is required, only one of the rest of the parameters in the square brackets is required in order for the Edit Command to work.
```

Comment:
Can we change a deadline task to a floating tasking with this format i.e. give an empty start date?